openshift-kube-apiserver/authorization/minimumkubeletversion/minimum_kubelet_version.go

```diff
@@ -0,0 +1,86 @@
+package minimumkubeletversion
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/blang/semver/v4"
+	nodelib "github.com/openshift/library-go/pkg/config/node"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+	corev1informers "k8s.io/client-go/informers/core/v1"
+	api "k8s.io/kubernetes/pkg/apis/core"
+	"k8s.io/kubernetes/pkg/auth/nodeidentifier"
+)
+
+var minimumKubeletVersion *semver.Version
+
+func SetMinimumKubeletVersion(version string) {
+	if version == "" {
+		return
+	}
```

Comment:
```suggestion
```

IMO, if we don't want to set a minimum kubelet version then this function shouldn't be called. A panic from MustParse would be clearer than a quiet no-op.